lxd/containers.go

```diff
@@ -356,7 +362,15 @@ func createFromImage(d *Daemon, req *containerPostReq) Response {
 			}
 
 			if !c.isPrivileged() {
-				return shiftRootfs(c, name, d)
+				err = shiftRootfs(c, name, d)
```

Comment:
Yeah, there are a few places where we run near-identical code in two cases. Definitely something we should spend some time on before it gets out of hand and we start getting odd bugs because we forgot to update some of the code paths.